src/fontra/backends/designspace.py

```diff
@@ -10,7 +10,10 @@
 import watchfiles
 from ..core.classes import VariableGlyph, StaticGlyph, Source, Layer
 from ..core.packedpath import PackedPathPointPen
+from typing import TYPE_CHECKING
 
+if TYPE_CHECKING:
+    from fontTools.ufoLib.glifLib import GlyphSet
```

Comment:
This should probably be an unconditional import, given the test failure.

Reply:
Oh duh. I forgot the most important statement!

Reply:
Why not import GlyphSet unconditionally, and forget about the `__future__` and `typing` imports?

Reply:
I guess. The future import has performance implications, since it postpones annotation evaluation and is slated to become the default at some point in time.